src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -1053,10 +1053,12 @@
 			v = "AdvancedVectorExtensions"
 		elseif x == "AVX2" and _ACTION > "vs2012" then
 			v = "AdvancedVectorExtensions2"
-		elseif x == "SSE2" then
-			v = "StreamingSIMDExtensions2"
-		elseif x == "SSE" then
-			v = "StreamingSIMDExtensions"
+		elseif cfg.architecture ~= "x64" then
```

Comment:
I was going to pull this, but a recent change has renamed x32->x86 and x64->x86_64
This needs to be updated to x86_64.

I also wonder if it should be `== "x86"` rather than `~= "x86_64"`, since it's precisely one architecture that supports this, not 'anything other than x86_64'.
Is there a possible 'unspecified' case for architecture that the ~= also catches?